### PR TITLE
Add `set_json_payload` method to TestRequest

### PR DESCRIPTION
**Pull request by boustrophedon**
Implements #824 - Add a `TestRequest::set_json_payload` convenience method.

### CODE REVIEW

File: src/test.rs [hunk 1]
```diff
@@ -477,6 +478,14 @@ impl TestRequest {
         self
     }
 
+    /// Set request payload serialized to JSON
+    pub fn set_json_payload<T: Serialize>(mut self, data: &T) -> Self {
```
Comment (fafhrd91) [inline src/test.rs]: let's name it `set_json`

**Review by fafhrd91 — APPROVED** [review]
could you also add entry to changes

**Comment by boustrophedon** [comment]
I updated CHANGES.md, added a line in the comment about setting the Content-Type header, and added a line to the test to assert the content type is application/json.

**Comment by boustrophedon** [comment]
The travis build failed only on the beta build with a spurious error:
```
After this operation, 38.1 MB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  libssl-dev libssl1.0.0 openssl
E: There were unauthenticated packages and -y was used without --allow-unauthenticated
The command "sudo apt-get install -y openssl libssl-dev libelf-dev libdw-dev cmake gcc binutils-dev libiberty-dev" failed and exited with 100 during .
```

Can you rerun?

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
thanks!

**Pull request merged**